OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -838,40 +1108,391 @@ namespace OloEngine
             if (!file.is_open())
                 return false;
 
-            // CSV header
-            file << "Index,Type,DrawKey,ViewportID,ViewLayer,RenderMode,MaterialID,ShaderID,Depth,Static,GroupID,DebugName\n";
+            file << "Index,Type,DrawKey,ViewportID,ViewLayer,RenderMode,MaterialID,ShaderID,Depth,Static,GroupID,DebugName,GpuTimeMs\n";
+
+            const auto& commands = !frame->PostSortCommands.empty() ? frame->PostSortCommands : frame->PreSortCommands;
```

Clang (14.0.6)

[warning] conditional operator with identical true and false expressions

(bugprone-branch-clone)